requirements.txt

```diff
@@ -9,4 +9,4 @@ natsort
 signac>=1.0.0
 watchdog
 webassets>=2.0.0
-werkzeug>=1.0.0
+werkzeug>=2.0.0
```

Comment:
We probably don't need to specify `werkzeug` here anymore. I think this was pinned because of a previous bug in werkzeug but flask's requirements should fetch a compatible version now. There have been new major releases of both packages since then.

Reply:
See https://github.com/glotzerlab/signac-dashboard/pull/78